js/pages/myToDoList.js

```diff
@@ -512,17 +525,17 @@ const renderMainBody = (data, collections, tab) => {
                                 
                                 started = true;
                                 
-                                template += `<li style="width:100%; margin:auto; margin-bottom:20px; border:1px solid lightgrey; border-radius:5px;">
-                                                <div class="row">
+                                template += `<li role="listitem" style="width:95%; margin:auto; margin-bottom:20px; border:1px solid lightgrey; border-radius:5px;">
```

Comment:
Similarly, is it best practice to specify this role when it's already the default role for li?

Reply:
Responding to both of your comments here: Thanks for this call-out. Testing both ways, I found the screen reader provides slightly more spoken information about the list when these tags are included. I think it's because this list was defined with a combination of li items and div tags (modules 1-4 have an inner list that were defined with div tags).